lib/tidewave/router.ex

```diff
@@ -99,8 +107,7 @@ defmodule Tidewave.Router do
         shell(port, conn)
 
       {^port, {:exit_status, status}} ->
-        {:ok, conn} = chunk(conn, "\nTIDEWAVE STATUS: #{status}")
-        {:ok, conn} = chunk(conn, "")
+        {:ok, conn} = chunk(conn, ~s|\n<tidewave_done>{"status":#{status}}|)
```

Comment:
We could also encode each chunk as JSON, which would help us distinguish about stderr and stdout… but unfortunately has the issue that we may have binary being send through which we cannot encode as JSON?

Reply:
Another option is to prepend the byte 1 to every message. This way we can add new messages format in the future. The last message we prepend the byte 255.

Reply:
Technically byte 1 and byte 255 could be in the output though?

Also, there is no guarantee that the chunks that we receive match the chunks sent by the server. So perhaps we should do our own framing on top? We could send each chunk as `<<type::8, size::32, data::binary-size(size)>>` and then on the client we can reconstruct the chunks. `data` would be raw bytes for output and json for exit type.